src/Compilers/CSharp/Test/Symbol/Symbols/MethodImplementationFlagsTests.cs

```diff
@@ -0,0 +1,189 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public class MethodImplementationFlagsTests : CSharpTestBase
+    {
+        [Fact]
+        public void TestInliningFlags()
+        {
+            var src = @"
+using System.Runtime.CompilerServices;
+
+public class C
+{
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public void M_Aggressive()
+    {
+    }
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public void M_NoInlining()
+    {
+    }
+}
+";
+
+            Action<ModuleSymbol> validator = module =>
+            {
+                var c = module.GlobalNamespace.GetMember<NamedTypeSymbol>("C");
+                var aggressiveInliningMethod = c.GetMember<MethodSymbol>("M_Aggressive");
+                Assert.Equal(MethodImplAttributes.AggressiveInlining, aggressiveInliningMethod.MethodImplementationFlags);
```

Comment:
>aggressiveInliningMethod.MethodImplementationFlags [](start = 70, length = 50)

It looks like this test doesn't actually test the added public API. Rename of the internal API made it hard to spot. I assume the other tests in this file have the same issue.